### PR TITLE
Make resteasy read and commit timeout configurable

### PR DESCRIPTION
**Pull request by schmitzhermes**
**What this PR does / why we need it**:
This PR makes the `read` and `connect` timeout of the Resteasy client configurable. 

**Special notes for your reviewer**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [x] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

**Review by jkroepke — CHANGES_REQUESTED** [review]
Hi, thanks for your contribution!

I'm ready using java.time.Duration as timeout parameter. https://github.com/adorsys/keycloak-config-cli/blob/2fe3664b68b50deb63904ace9d539b1923b0af9a/src/main/java/de/adorsys/keycloak/config/properties/KeycloakConfigProperties.java#L142

For Consistency reasons, I prefer Duration over Interger here, too.

**Comment by jkroepke** [comment]
Additionally, please create a test for this. You can use something like this 

https://github.com/adorsys/keycloak-config-cli/blob/2fe3664b68b50deb63904ace9d539b1923b0af9a/src/test/java/de/adorsys/keycloak/config/provider/KeycloakProviderIT.java#L40-L56

as reference.

Background: This repository has automatic dependencies enabled including major bumps. By having a test for this functionality,  I just know that the feature still works even after a upgrade of resteasy.

**Comment by schmitzhermes** [comment]
Thanks for your review. 9c58 and 794f should fix your comments. 

By the way: Thanks for your amazing work - it's a real pleasure to work with the library!

File: src/test/java/de/adorsys/keycloak/config/provider/KeycloakProviderIT.java [hunk 1]
```diff
@@ -22,20 +22,62 @@
...
+    class ResteasyConnectTimeout extends AbstractImportTest {
+        @Test
+        @Timeout(value = 1L)
+        void run() {
+            // connect timeout since IP is not reachable - test fails if it exceeds one second which in turn shows that
+            // the configuration is applied
+            ProcessingException thrown = assertThrows(ProcessingException.class, keycloakProvider::getInstance);
+            assertNotNull(thrown.getCause());
+            assertTrue(thrown.getCause() instanceof ConnectTimeoutException);
+            assertThat(thrown.getCause().getMessage(), containsString("Connect timed out"));
```
Comment (jkroepke) [inline src/test/java/de/adorsys/keycloak/config/provider/KeycloakProviderIT.java:77]: Looks like java 11 and java 17 have different messages. I guess a case insensitive check would be the best here. Or just remove the C.

**Review by jkroepke — APPROVED** [review]
Thanks a lot for your changes. I add a comment to fix the failing tests.

File: src/test/java/de/adorsys/keycloak/config/provider/KeycloakProviderIT.java (hunk 1 above)
Comment (schmitzhermes) [reply src/test/java/de/adorsys/keycloak/config/provider/KeycloakProviderIT.java]: weird :-) fixed

**Comment by schmitzhermes** [comment]
Hi @jkroepke - as I need this feature in my setup, do u already know when the next release will take place? Just so I can plan it accordingly.

**Comment by jkroepke** [comment]
4.5.0 is 28 days old, I'll create a new one.

Enjoy https://github.com/adorsys/keycloak-config-cli/releases/tag/v4.6.0

**Pull request merged**